## 2. Continuing exercise 1, find the Wald, Wilson, and Agresti-Couli 95% confidence intervals for $\pi$ (the probability of success in each experiment. Interpret the intervals in the content of the experiments. (See section 1.3.4 in your textbook for this material)

Infant and Cheerios.
w= 9 successes are observed out of n= 20 trials. 

In [7]:
install.packages("binom")
library(binom)
w_a<-9
n_a<- 20
alpha<-0.05
binom.confint(x=w_a,n=n_a,conf.level = 1 - alpha, methods =c("asymptotic","wilson","ac"))

Updating HTML index of packages in '.Library'
Making 'packages.html' ... done


method        x n  mean lower     upper    
1 agresti-coull 9 20 0.45 0.2580617 0.6580508
2 asymptotic    9 20 0.45 0.2319678 0.6680322
3 wilson        9 20 0.45 0.2581979 0.6579147

*Agretsi-Coull CI*: We are 95% confident that the population proportion of Cheerios that made it into an infant's mouth is between 0.258 and 0.658

*Wald CI*: We are 95% confident that the population proportion of Cheerios that made it into an infant's mouth is  between 0.232 and 0.668

*Wilson CI*: We are 95% confident that the population proportion of Cheerios that made it into an infant's mouth is between 0.258 and 0.658

### 2B. Billiard Games
w= 15 ball sunks are observed out of the n=25 consecutive breaks. 

In [8]:
w_b<-15
n_b<- 25
alpha<-0.05
binom.confint(x=w_b,n=n_b,conf.level = 1 - alpha, methods = c("asymptotic","wilson","ac"))

method        x  n  mean lower     upper    
1 agresti-coull 15 25 0.6  0.4069662 0.7663953
2 asymptotic    15 25 0.6  0.4079635 0.7920365
3 wilson        15 25 0.6  0.4073946 0.7659670

*Agretsi-Coull CI*: We are 95% confident that the true (population) proportion of a person's success of sinking a ball on the break is between 0.407 and 0.766

*Wald CI*: We are 95% confident that the true (population) proportion of a person's success of sinking a ball on the break is between 0.408 and 0.792

*Wilson CI*: We are 95% confident that the true (population) proportion of a person's success of sinking a ball on the break is between 0.407 and 0.766

### 2C. Germination Rate.

w= 48 seeds sprouted out of the n=64 seeds. 

In [9]:
w_c<-48
n_c<- 64
alpha<-0.05
binom.confint(x=w_c,n=n_c,conf.level = 1 - alpha, methods =c("asymptotic","wilson","ac"))

method        x  n  mean lower     upper    
1 agresti-coull 48 64 0.75 0.6309324 0.8407556
2 asymptotic    48 64 0.75 0.6439138 0.8560862
3 wilson        48 64 0.75 0.6318372 0.8398507

*Agretsi-Coull CI*: We are 95% confident that the true (population) proportion of a particular type of sweet corn seeds that will sprout after being planted into soil is between 0.631 and 0.841

*Wald CI*: We are 95% confident that the true (population) proportion of of a particular type of sweet corn seeds that will sprout after being planted into the soil is between 0.644 and 0.856

*Wilson CI*: We are 95% confident that the true (population) proportion of of a particular type of sweet cornsweet corn seeds that will sprout after being planted into the soil is between 0.632 and 0.840

### 2D.  Alternative Fuel Cars.

w= 14 alternative fuel cars are observed out of n=125 cars that passed through the intersection. 

This is a Poisson random variable being that the PRV counts the number of given occurences in a given interval.
The two maint characteristics of the PVR:

1)Rate of occurence is assumed to be the same throughout the interval.

2)Occurences are independent of one another. 

We can still use the normal approximation to compute the CI.

In [3]:
rate<-14
var.poisson<-rate
lower<-rate-qnorm(p=1-alpha/2)*sqrt(var.poisson)
upper<-rate+qnorm(p=1-alpha/2)*sqrt(var.poisson)
round(data.frame(lower,upper),4)

lower  upper  
1 6.6665 21.3335

I am 95% sure that the true rate per half hour of alternative fuel cars in any random sample of cars that is collected in this location at a similar time of day and day of week is between 6.7 and 21.3 cars per half hour. 